flask_server.py

```diff
@@ -21,7 +21,8 @@ def get_log_file_list():
     if not os.path.exists(scraping_logs_path):
         return []
     with open(scraping_logs_path, "r+") as o:
-        return [log_lines for log_lines in o.readlines()]
+        logs = [log_lines for log_lines in o.readlines()]
```

Comment:
Maybe consider log_lines for log_lines in reversed(o.readlines()) so you dont need an extra line or o.readlines()[::-1]